sensor/common/sensor/sensor.go

```diff
@@ -167,6 +169,15 @@ func (s *Sensor) Start() {
 		customRoutes = append(customRoutes, koCacheRoute)
 	}
 
+	// Enable endpoint to retrieve vulnerability definitions if local image scanning is enabled.
+	if features.LocalImageScanning.Enabled() {
+		route, err := newScannerDefinitionsRoute(s.centralEndpoint)
+		if err != nil {
+			log.Panicf("error creating scanner definition route: %v", err)
```

Comment:
I think it may be better to do something similar to what we do for the kocache above this: `utils.Should(errors.Wrap(err, "Failed to create scanner definitions route"))`

That way we do not panic in production environments if we cannot serve scanner definitions from Sensor